cmd/container-disk-v1alpha/entry-point.sh

```diff
@@ -26,13 +26,16 @@ if [ -z "$COPY_PATH" ]; then
 	exit 1
 fi
 
-if [ ! -f "$IMAGE_PATH" ]; then
+if [ ! -v IMAGE_PATH ]; then
```

Comment:
why didn't the previous way work?  If the IMAGE_PATH file exists, then this logic is supposed to use the custom path, otherwise attempt to detect the file.